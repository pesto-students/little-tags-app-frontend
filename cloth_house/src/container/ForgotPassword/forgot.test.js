import createComponentWithIntl from '../../utils/helper_snapshot'
import Forgot from "./index"

it('Forgot Password Snapshot', () => {
    const component = createComponentWithIntl(<Forgot />);
    let tree = component.toJSON()
  
    expect(tree).toMatchSnapshot()
  
  });