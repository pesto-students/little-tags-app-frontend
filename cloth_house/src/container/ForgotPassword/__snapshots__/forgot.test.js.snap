// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Forgot Password Snapshot 1`] = `
<p
  className="paddingTop25"
>
  <div
    className="ant-alert ant-alert-info ant-alert-no-icon"
    data-show={true}
    role="alert"
    style={Object {}}
  >
    <div
      className="ant-alert-content"
    >
      <div
        className="ant-alert-message"
      />
      <div
        className="ant-alert-description"
      />
    </div>
  </div>
</p>
`;
